File: Modul 4/Bahasa C/PRAK402-2310817120001-DinaIzzatiElfadheya.c
#include <stdio.h>
int main() {
    int bilangan;
    
    printf("Input\n");
    scanf("%d", &bilangan );
    
    printf("\nOutput\n");
    
    for (int i = 1; i <= bilangan; i += 2) { // bilangan ganjil
        printf("%d ", i);
    }
    printf("\n");//bilangan genap
    for (int i = (bilangan % 2 == 0 ? bilangan : bilangan - 1); i >= 2; i -= 2) {
        printf("%d ", i);
    }
    printf("\n");
    return 0;
}

File: Modul 4/Bahasa Python/PRAK402-2310817120001-DinaIzzatiElfadheya.py
bilangan = int(input("Input\n"))
print("\nOutput")

for i in range(1, bilangan + 1, 2): #bilangan ganjil 
    print(i, end=" ")
print()
if bilangan % 2 == 0:
    for i in range(bilangan, 1, -2): #bilangan genap
        print(i, end=" ")
else:
    for i in range(bilangan - 1, 1, -2):
        print(i, end=" ")
print()